docs/help/_posts/2010-04-01-argument-matchers.markdown

```diff
@@ -98,8 +102,32 @@ calculator.Received().Add(Arg.Is(0), Arg.Any<int>());
 
 This matcher normally isn't required; most of the time we can just use `0` instead of `Arg.Is(0)`. In some cases though, NSubstitute can't work out which matcher applies to which argument (arg matchers are actually fuzzily matched; not passed directly to the function call). In these cases it will throw an `AmbiguousArgumentsException` and ask you to specify one or more additional argument matchers. In some cases you may have to explicitly use argument matchers for every argument.
 
+## Matching `out` and `ref` args
+
+Argument matchers can also be used with `out` and `ref` (NSubstitute 4.0 and later with C# 7.0 and later).
+
+{% examplecode csharp %}
+calculator
+    .LoadMemory(1, out Arg.Any<int>())
+    .Returns(x => {
+        x[1] = 42;
+        return true;
+    });
+
+int memoryValue;
+var hasEntry = calculator.LoadMemory(1, out memoryValue);
```

Comment:
Should be use `out var memory` or `out int memory` syntax here? It will promote new language feature and will look more consistent with `out Arg.Any<>()` we have above 😉

Reply:
Good catch! Updated. 👍